loaders/src/glTF/2.0/Extensions/KHR_texture_basisu.ts

```diff
@@ -34,7 +33,8 @@ export class KHR_texture_basisu implements IGLTFLoaderExtension {
 
     /** @hidden */
     public loadTextureInfoAsync(context: string, textureInfo: ITextureInfo, assign: (babylonTexture: BaseTexture) => void): Nullable<Promise<BaseTexture>> {
-        this._textureInfo = textureInfo; // need to save this for use in _loadTextureAsync below
+        const texture = ArrayItem.Get(`${context}/index`, this._loader.gltf.textures, textureInfo.index);
+        texture._textureInfo = textureInfo; // need to save this for use in _loadTextureAsync below

```

Comment:
You probably want to do this in the core loader and not just for this extension.

Reply:
True, corrected!